.github/workflows/spark-periodic-tests.yml

```diff
@@ -0,0 +1,58 @@
+name: Spark Periodic Integration Tests
```

Comment:
Curious why we separated this from the other periodic tests?

Reply:
I did this originally because of the different schedule, the spark tests run once a week vs every day (due to machine cost of running the databricks tests.)

Reply:
ah I see. What's the cost for each run? I wonder if we should do twice a week at least (eg. tuesday and friday)